packages/flutter_driver/lib/src/driver/driver.dart

```diff
@@ -169,16 +164,10 @@ class FlutterDriver {
   /// [logCommunicationToFile] determines whether the command communication
   /// between the test and the app should be logged to `flutter_driver_commands.log`.
   ///
-  /// [FlutterDriver] multiplies all command timeouts by [timeoutMultiplier].
-  ///
   /// [isolateNumber] (optional) determines the specific isolate to connect to.
   /// If this is left as `null`, will connect to the first isolate found
   /// running on [dartVmServiceUrl].
   ///
-  /// [isolateReadyTimeout] determines how long after we connect to the VM
-  /// service we will wait for the first isolate to become runnable. Explicitly
-  /// specified non-null values are not affected by [timeoutMultiplier].
-  ///
```

Comment:
Add a documentation block about when the completed future returns and the fact that it explicitly will not time out, so if a developer wants that, they need to do it explicitly.

Reply:
done